Makefile

```diff
@@ -123,7 +123,7 @@ ci-release: init-package
 build-examples: ## Build all of the example packages
 	@test -s $(ZARF_BIN) || $(MAKE) build-cli
 
-	@test -s ./build/zarf-package-dos-games-$(ARCH).tar.zst || $(ZARF_BIN) package create examples/game -o build -a $(ARCH) --confirm
+	@test -s ./build/zarf-package-dos-games-$(ARCH).tar.zst || $(ZARF_BIN) package create examples/doom -o build -a $(ARCH) --confirm
```

Comment:
was there a reason to change this from `game` -> `doom`? It originally was `doom` iirc, but we changed it when the package started installing multiple dos games vs just doom.

Reply:
Synching the path and the title of the page to make the menu sorting correct. We can change it the other way as well, but the menu wasn't being sorted correctly.

Reply:
@YrrepNoj iirc you can do sorting by prefixing with numbers right? Would like to get this PR finished pretty quickly as it will ease our documentation syncing.

Reply:
Docusaurus uses number that are prepended to folders/files to determine the order in menu bars.

`docs/6-developer-guide/2-testing.md`

Reply:
Sure, I can change it to games. I changed it to match the title in the documentation. I can change both to DOS Games.

Reply:
Changed the title of the README from "Doom in the Cluster" to "DOS Games in the Cluster" and renamed the path appropriately. There are other references to Doom that exist in the docs that may need to be updated.